client/my-sites/stats/stats-card-upsell/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { isEnabled } from '@automattic/calypso-config';
+import page from '@automattic/calypso-router';
+import { Button, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import './style.scss';
+
+interface Props {
+	className: string;
+	statType: string;
+	siteId: number;
+}
+
+const StatsCardUpsell: React.FC< Props > = ( { className, statType, siteId } ) => {
+	const translate = useTranslate();
+
+	const onClick = ( event: React.MouseEvent< HTMLButtonElement, MouseEvent > ) => {
+		event.preventDefault();
+
+		const source = isEnabled( 'is_running_in_jetpack_site' ) ? 'jetpack' : 'calypso';
```

Comment:
Would it be more consistent if we added this?
```suggestion
        const isOdyssey = isEnabled( 'is_running_in_jetpack_site' );
		const source = isOdyssey ? 'jetpack' : 'calypso';
```

Also I'm wondering if `isEnabled( 'is_running_in_jetpack_site' )` will always mean it's Odyssey cc: @Automattic/red